src/main/scala/org/scalajs/tools/tsimporter/sc/Definitions.scala

```diff
@@ -51,11 +52,16 @@ object QualifiedName {
   val scala = Root dot Name.scala
   val scala_js = scala dot Name.scalajs dot Name.js
   val java_lang = Root dot Name.java dot Name.lang
+  val js = Root dot Name.js
+  val jstypedarray = Root dot Name.js dot Name.typedarray
 
   val Array = scala_js dot Name("Array")
   val Dictionary = scala_js dot Name("Dictionary")
   val FunctionBase = scala_js dot Name("Function")
   val Object = scala_js dot Name("Object")
+  val Thenable = js dot Name("Thenable")
+  val JsArray = js dot Name("Array")
```

Comment:
Please use the casing `JSArray` instead of `JsArray`, to match our existing coding conventions (not sure whether we've written it down somewhere, so that's on me).